wms_connector/views/stock_warehouse.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Akretion
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="stock_warehouse_form_view">
+        <field name="name">stock.warehouse.form (in wms_connector)</field>
+        <field name="model">stock.warehouse</field>
+        <field name="inherit_id" ref="stock.view_warehouse" />
+        <field name="arch" type="xml">
+            <field name="code" position="after">
+                <field name="active_wms_sync" />
+            </field>
+            <xpath expr="//notebook" position="inside">
+                <page
+                    name="wms_sync"
+                    string="Synchronization"
+                    attrs="{'invisible': [('active_wms_sync', '=', False)]}"
+                >
+                    <group>
+                        <field name="wms_export_task_id" />
+                        <field name="wms_import_confirm_delivery_task_id" />
+                        <field name="wms_import_confirm_reception_task_id" />
+                    </group>
+                    <group>
+                        <field name="wms_export_cron_id" />
+                        <field name="wms_import_confirm_delivery_cron_id" />
+                        <field name="wms_import_confirm_reception_cron_id" />
+                    </group>
+                    <group>
+                        <field name="wms_product_sync_filter_id" />
+                        <field name="wms_export_product_filter_id" />
+                        <field name="wms_export_picking_in_filter_id" />
+                        <field name="wms_export_picking_out_filter_id" />
+                    </group>
+                    <group>
+                        <button
+                            name="refresh_wms_products"
+                            type="object"
+                            string="Refresh WMS products"
+                        />
+                        <field name="wms_product_sync_ids">
```

Comment:
Add a smart button that open the view in a dedicated view. Having a O2M integrated in the view with a lot of line is not usable